cspell.json

```diff
@@ -46,6 +47,8 @@
     "Alderaan",
     "Tatooine",
     "astromech",
+    "geocode",
+    "structs",
```

Comment:
cspell have some issue with some dictionary marked as applicable on JS but not on TS.
For example:
https://github.com/streetsidesoftware/cspell-dicts/blob/6306b1d11d32e4f66721053d5fccf119ca7ce26e/dictionaries/fullstack/fullstack.txt#L46
https://github.com/streetsidesoftware/cspell-dicts/blob/6306b1d11d32e4f66721053d5fccf119ca7ce26e/dictionaries/fullstack/cspell-ext.json#L23

@saihaj Can you please contribute fixes to cspell?